moped-editor/src/views/projects/projectView/ProjectComponents/mapStyleSettings.js

```diff
@@ -14,6 +14,7 @@ export const COLORS = {
   blueLight: "#a1cdf7",
   steel: "#607d8f",
   white: "#fff",
+  test: "#f28100",
```

Comment:
Leaving this here for now to make edit mode obvious, but I'll change the key name and hex color when we're set on something.

Reply:
Backlog issue sounds good to me. I might try to squeeze this in the edit drawn PR that I'll open tomorrow. I updated the color in that one, and it looks great 😎 Love Coolers btw

Reply:
👍 https://github.com/cityofaustin/atd-data-tech/issues/10992